src/plugins/vis_type_timeseries/public/application/components/aggs/cumulative_sum.js

```diff
@@ -93,6 +95,7 @@ export function CumulativeSumAgg(props) {
 CumulativeSumAgg.propTypes = {
   disableDelete: PropTypes.bool,
   fields: PropTypes.object,
+  indexPatten: PropTypes.string,
```

Comment:
There is a typo here and on some other places `indexPatten` should be `indexPattern`

Reply:
thank you, nice catch